src/HackerNews/SelectedNewStore.ts

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import { NewsDataResponse } from './Interfaces';
+
+interface NewStore {
+  selectedNew: React.ComponentState;
+  setSelectedNewCallback: (event: React.MouseEvent<HTMLAnchorElement>, obj: NewsDataResponse) => void;
```

Comment:
Обычно наименование коллбэков с префиксом `on` идет, ты наверно видел такое в тех же `<button>` и тд.
Поэтому тут можно просто что то типо
```suggestion
  onSelectNews: (event: React.MouseEvent<HTMLAnchorElement>, obj: NewsDataResponse) => void;
```